common/src/main/java/tachyon/security/UserInformation.java

```diff
@@ -40,11 +48,35 @@
   private static final boolean IS_64_BIT = PlatformUtils.PROCESSOR_BIT.contains("64");
   private static final boolean AIX = PlatformUtils.OS_NAME.equals("AIX");
 
+  /** The configuration to use */
+  private static TachyonConf sConf;
+  /** The authentication method to use */
+  private static AuthenticationFactory.AuthTypes sAuthType;
```

Comment:
This is actually related to a closed PR, but can you rename `AuthTypes` to `AuthType`. In java the convention is to use singular for enum, it's quite confusing the other way.